src/NUnitConsole/nunit3-console.tests/ConsoleRunnerTests.cs

```diff
@@ -1,32 +1,58 @@
 ﻿using System;
 using System.IO;
 using System.Xml;
+using NSubstitute;
 using NUnit.Common;
 using NUnit.Engine;
 using NUnit.Engine.Extensibility;
 using NUnit.Engine.Services;
 using NUnit.Framework;
+using NUnit.Framework.Internal;
 
 namespace NUnit.ConsoleRunner.Tests
 {
     class ConsoleRunnerTests
     {
-        [Test, Ignore("Needs to be rewritten")]
```

Comment:
This is unrelated, but I wanted to add NSubstitute for my tests, so put this one back while I was here.